.github/workflows/ci.yml

```diff
@@ -435,9 +437,19 @@ jobs:
           export MAVEN_OPTS="${MAVEN_INSTALL_OPTS}"
           $RETRY $MAVEN install ${MAVEN_FAST_INSTALL} -am -pl "${{ matrix.modules }}"
       - name: Maven Tests
-        if: matrix.modules != 'plugin/trino-memsql'
+        if: matrix.modules != 'plugin/trino-memsql' && matrix.modules != 'plugin/trino-delta-lake'
         run: $MAVEN test ${MAVEN_TEST} -pl ${{ matrix.modules }} ${{ matrix.profile != '' && format('-P {0}', matrix.profile) || '' }}
       # Additional tests for selected modules
+      - name: Delta Lake Tests
+        env:
+          ABFS_CONTAINER: ${{ secrets.AZURE_ABFS_CONTAINER }}
+          ABFS_ACCOUNT: ${{ secrets.AZURE_ABFS_ACCOUNT }}
+          ABFS_ACCESS_KEY: ${{ secrets.AZURE_ABFS_ACCESS_KEY }}
+        if: matrix.modules == 'plugin/trino-delta-lake'
+        # TODO: Restore condition above: && env.ABFS_ACCOUNT != '' && env.ABFS_CONTAINER != '' && env.ABFS_ACCESS_KEY != ''
+        run: |
+          # TODO: Split this onto multiple lines
+          $MAVEN test ${MAVEN_TEST} -pl :trino-delta-lake -Dhive.hadoop2.azure-abfs-container="${ABFS_CONTAINER:?ABFS container empty or unset}" -Dhive.hadoop2.azure-abfs-account="${ABFS_ACCOUNT:?ABFS account empty or unset}" -Dhive.hadoop2.azure-abfs-access-key="${ABFS_ACCESS_KEY:?ABFS access key empty or unset}"
```

Comment:
per `TODO: Split this onto multiple lines` -- with `run: |` you should be able
to format the value as

```
some long \
  command \
  wtih many long \
  arguments
```

cc @nineinchnick

Reply:
Yeah, I just did it this way because I wanted to be 100% sure it would work the first time so I wouldn't have to wait for CI twice.